salt/cloud/clouds/gogrid.py

```diff
@@ -82,6 +82,12 @@ def create(vm_):
     '''
     Create a single VM from a data dict
     '''
+
+    # Since using "provider: <provider-engine>" is deprecated, alias provider
+    # to use driver: "driver: <provider-engine>"
+    if 'provider' in vm_:
+        vm_['driver'] = vm_.pop('provider')
```

Comment:
This is just a stop-gap change until `provider` is completely deprecated in favor of `driver`. The `vm_` dict gets populated with `driver` in `cloud/__initi__.py`. I missed the gogrid driver in #24796.